### PR TITLE
Adds space rattlesnakes to the game

### PR DESCRIPTION
**Pull request by colossusqw**
[LABEL][feature][sprites]

## About the PR 
This PR adds rattlesnakes as random pest events, they spawn one rattlesnake with a rare chance of two spawning. The rattlesnakes are not aggressive, however getting close to them will cause them to coil up and rattle at you, and getting very close will make them angry at you.

## Why's this needed? 
Adding more variety to the random pest event, this time a critter that wont attack unless you get very dlose, but that provides a serious fight if you ignore the warnings.

## Changelog

```changelog
(u)Hans
(*)Added space rattlesnakes to the game. Beware!
```

### CODE REVIEW

**Comment by Conchuckter** [comment]
Do they inject reagents?

**Comment by colossusqw** [comment]
i forgot some images to show them off
you fucked up
image
image
image

File: code/obj/critter/misc.dm [hunk 1]
```diff
@@ -519,6 +519,73 @@
...
+				src.oldtarget_name = C.name
+				src.visible_message("<span class='combat'><b>[src]</b> charges at [C.name]!</span>")
+				src.task = "chasing"
+				break
+
+	ChaseAttack(mob/M)
+		..()
+		if(!ON_COOLDOWN(src, "snake bite", 10 SECONDS))
+			M.visible_message("<span class='combat'><B>[src]</B> bites [src.target]!</span>")
+			M.reagents?.add_reagent("viper_venom", 40)
```
Comment (Tarmunora) [inline code/obj/critter/misc.dm:584]: seems excessive

**Review by Tarmunora — CHANGES_REQUESTED** [review]

File: code/obj/critter/misc.dm (hunk 1 above)
Comment (colossusqw) [reply code/obj/critter/misc.dm:584]: oh, i changed it on the regular attack but i've forgotten to change it here, apologies, gonna make it 10

**Comment by pali6** [comment]
Feels perhaps a bit close to scorpions / wasps I think? Also very weird that it's separate from `/obj/critter/snake` in the type hierarchy. Perhaps modify `/obj/critter/snake` to be their common parent and make your thing `/obj/critter/rattlesnake` and the current snake `/obj/critter/snake/wizard` or something like that.

**Comment by Tarmunora** [comment]
bloodloss might be a bit too high but we'll see :lazy:

**Pull request merged**